{{cookiecutter.project_name}}/.readthedocs.yml

```diff
@@ -0,0 +1,23 @@
+# Read the Docs configuration file
```

Comment:
Thanks a lot!

Do we really need this file? It works for me without it.
Because these are defaults

Reply:
As I told in PR's description, readthedocs would find and use `/requirements.txt` first instead of `/docs/requirements.txt` if it's present in repo. If you use and store `/requirements.txt` for backwards compatibility with some tools (for example I use it for Heroku Python Buildpack in [scdlbot](https://github.com/gpchelkin/scdlbot)), you should set the requirements file explicitly. Maybe it is not _really_ needed.